src/pages/create.tsx

```diff
@@ -0,0 +1,161 @@
+import { useState } from "react";
+import { useAuth } from "../../utils/firebase/auth";
+import { Text, Button, Input } from "../components/atomic";
+import { createProgram } from "../generated/graphql";
+import TitledInput from "../components/TitledInput";
+
+const BlobCircle = () => {
+  const sizes = "h-24 w-24 md:h-64 md:w-64 lg:h-80 lg:w-80";
+  return (
+    <div
+      className={`${sizes} rounded-full bg-skyblue overflow-hidden flex justify-center items-end pointer-events-none`}
+    >
+      <img src="/static/HappyBlobs.svg" className="w-11/12 select-none" />
+    </div>
+  );
+};
+
+const SignUpPage = () => {
+  const [stage, setStage] = useState(0);
+  const [programName, setProgramName] = useState("");
+  const [programLogo, setProgramLogo] = useState("");
+  const [programIdentifier, setProgramIdentifier] = useState("");
+  const [displayError, setError] = useState("");
+
+  const stepOne = () => {
+    return (
+      <div className="flex flex-col w-full px-4 py-12 md:w-120">
+        <Text h1 b>
+          Create your mentorship!
+        </Text>
+        <div className="h-6" />
+
+        <div className="flex w-full">
+          <TitledInput
+            title="What is the name of your mentorship program?"
+            name="Program Name"
+            placeholder="e.g. Nova Mentorship"
+            value={programName}
+            className="flex-1"
+            onChange={(e) => {
+              setProgramName(e.target.value);
+            }}
+          />
+        </div>
+        <div className="h-3" />
+
+        <Button
+          disabled={programName.length == 0}
+          onClick={() => {
+            if (programName) {
+              setStage(stage + 1);
+              setError("");
+            } else setError("Please enter a program name.");
+          }}
+        >
+          Next
+        </Button>
+        <div className="h-6" />
+
+        <Text className="text-error">
+          {displayError + "TODO: Proper error box"}
+        </Text>
+      </div>
+    );
+  };
+
+  const stepTwo = () => {
+    return (
+      <div className="flex flex-col w-full px-4 py-12 md:w-120">
+        <Text h1 b>
+          Program Details
+        </Text>
+        <Text p>
+          Let's set up some basic information! You can also do these later.
+        </Text>
+        <div className="h-6" />
+        <div className="">
+          <Text b>Logo</Text>
+          <div className="h-1" />
+          <Button variant="inverted" size="small">
+            Choose File
+          </Button>
+          <input // TODO: display preview of logo
```

Comment:
also todo: create a custom file input component?

Reply:
Yes-- talked to Kevin about this. Will create a new PR after enabling file upload in the backend